.changeset/tonic-ui-1057.md

```diff
@@ -0,0 +1,9 @@
+---
+"@tonic-ui/mcp": patch
+"@tonic-ui/react": patch
+"@tonic-ui/react-base": patch
+"@tonic-ui/styled-system": patch
+"@tonic-ui/utils": minor
+---
```

Comment:
Package | Release Type | Reason
:-- | :-- | :-
@tonic-ui/mcp | patch | Internal fixes, no new user-facing features
@tonic-ui/react | patch | Internal fixes, no new user-facing features
@tonic-ui/react-base | patch | Internal fixes, no new user-facing features
@tonic-ui/styled-system | patch | Now uses `utils.get` internally; behavior is backward-compatible
@tonic-ui/utils | minor | Adds lodash-compatible `get` function with bracket notation support (new feature)